src/pages/build-apps/guides/authentication.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Authentication
+description: Register and sign in users with identities on the Stacks blockchain
+experience: beginners
+tags:
+  - tutorial
+images:
+  large: /images/pages/write-smart-contracts.svg
+  sm: /images/pages/write-smart-contracts-sm.svg
+---
+
+## Introduction
+
+This guide explains how to authenticate users with the [`connect`](https://github.com/blockstack/ux/tree/master/packages/connect#stacksconnect) package of Stacks.js.
+
+Authentication provides a way for users to identify themselves to an app while retaining complete control over their credentials and personal details. It can be integrated alone or used in conjunction with [transaction signing](/build-apps/tutorials/transaction-signing) and [data storage](/build-apps/tutorials/data-storage), for which it is a prerequisite.
+
+Users who register for your app can subsequently authenticate to any other app with support for the [Blockchain Naming System](/build-apps/references/bns) and vice versa.
+
+[See the Todos app tutorial](/build-apps/tutorials/todos) for a concrete example of this functionality in practice.
```

Comment:
I'd use one of the inline links instead - it shows a preview. this would also apply to the otherguides